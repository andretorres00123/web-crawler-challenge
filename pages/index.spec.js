import React from 'react';
import renderer from 'react-test-renderer';
import Index from './index';

describe('Home page', () => {
   test('should renders the home page', () => {
      const tree = renderer.create(<Index />).toJSON()
      expect(tree).toMatchSnapshot()
    });
});
