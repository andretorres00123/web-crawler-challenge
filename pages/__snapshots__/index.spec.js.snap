// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home page should renders the home page 1`] = `
<div
  className="css-vurnku"
>
  <div
    className="css-1hwv0r3"
  >
    <div
      className="sc-AxjAm iNcNBE css-vurnku"
    >
      home-title
    </div>
    <div
      className="css-vurnku"
    >
       home-title-author
    </div>
  </div>
</div>
`;
